xml/System.Windows.Forms/Form.xml

```diff
@@ -6389,10 +6389,11 @@ This example assumes that the `CreateMyOpaqueForm` method is called from another
         <Parameter Name="updateContainerSize" Type="System.Boolean" Index="2" FrameworkAlternate="windowsdesktop-7.0" />
       </Parameters>
       <Docs>
-        <param name="xScaleFactor">To be added.</param>
-        <param name="yScaleFactor">To be added.</param>
-        <param name="updateContainerSize">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="xScaleFactor">The scale factor to be applied on width of the property being scaled.</param>
+        <param name="yScaleFactor">The scale factor to be applied on height of the property being scaled.</param>
+        <param name="updateContainerSize">
+          <see langword="true" /> to resize of the Form along with properties being scaled; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
          <see langword="true" /> to resize the Form along with properties being scaled; otherwise, <see langword="false" />.</param>
```

Reply:
👍